bootstrap.py

```
@@ -269,32 +269,51 @@ def _resolve_bot_client(platform_name: str) -> object | None:
         return None
 
     def _resolve_bot_self_id(platform_name: str) -> str:
-        """从已连接的反向 WebSocket 客户端中解析 bot 的 self_id（QQ 号）。
-
-        aiocqhttp 的 CQHttp 实例以 self_id 为键保存已连接的 API 客户端
-        （``_wsr_api_clients``）。只有连接中恰有一个非零 self_id 时才返回，
-        避免多 bot 平台把其他账号误写入合并转发节点；解析失败时返回空串，
-        由调用方保留 SDK 的默认值。
+        """从平台上所有已连接实例汇总 bot 的 self_id（QQ 号）。
```

Ruff (0.16.3)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)